engine-rest/engine-rest-openapi-generator/src/test/java/org/operaton/bpm/engine/rest/openapi/generator/impl/SchemaValidatorTest.java

```diff
@@ -0,0 +1,75 @@
+package org.operaton.bpm.engine.rest.openapi.generator.impl;
```

Comment:
License Header missing

Reply:
added.

BTW: Missing license headers are automatically added by the code cleanup action next day.